dnnv/_manage/linux/environment/dependencies/installers/common.py

```diff
@@ -142,10 +147,13 @@ def run(self, env: Environment, dependency: Dependency):
         if dependency.is_installed(env):
             return
 
+        openblas_url = (
+            "https://github.com/xianyi/OpenBLAS/archive/v{self.version}.tar.gz"
```

Comment:
This needs to be an f-string
```suggestion
            f"https://github.com/xianyi/OpenBLAS/archive/v{self.version}.tar.gz"
```